From a709a07bff91a7b2dd4e4a412090850a73937999 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Mon, 7 Mar 2016 13:36:32 -0500
Subject: [PATCH 2/3] data: switch back to Xorg by default

Consensus seems to be that switching to wayland for user sessions
by default is a bit premature.

This commit moves it back to Xorg by default.

We'll save wayland by default for 3.22

https://bugzilla.gnome.org/show_bug.cgi?id=763266
---
 data/Makefile.am                                         | 4 ++--
 data/{gnome-xorg.desktop.in => gnome-wayland.desktop.in} | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)
 rename data/{gnome-xorg.desktop.in => gnome-wayland.desktop.in} (85%)

diff --git a/data/Makefile.am b/data/Makefile.am
index ae08730..d0a7310 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -9,7 +9,7 @@ hwcompatdir = $(pkgdatadir)
 hwcompat_DATA = hardware-compatibility
 
 xsessiondir = $(datadir)/xsessions
-xsession_in_files = gnome.desktop.in gnome-xorg.desktop.in
+xsession_in_files = gnome.desktop.in
 
 if BUILD_SESSION_SELECTOR
 xsession_in_files += gnome-custom-session.desktop.in
@@ -18,7 +18,7 @@ endif
 xsession_DATA = $(xsession_in_files:.desktop.in=.desktop)
 
 wayland_sessiondir = $(datadir)/wayland-sessions
-wayland_session_in_files = gnome.desktop.in
+wayland_session_in_files = gnome-wayland.desktop.in
 wayland_session_DATA = $(wayland_session_in_files:.desktop.in=.desktop)
 
 sessiondir = $(datadir)/gnome-session/sessions
diff --git a/data/gnome-xorg.desktop.in b/data/gnome-wayland.desktop.in
similarity index 85%
rename from data/gnome-xorg.desktop.in
rename to data/gnome-wayland.desktop.in
index d00b3dc..852be87 100644
--- a/data/gnome-xorg.desktop.in
+++ b/data/gnome-wayland.desktop.in
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=GNOME on Xorg
+_Name=GNOME on Wayland
 _Comment=This session logs you into GNOME
 Exec=gnome-session
 TryExec=gnome-session
-- 
2.10.1

